### PR TITLE
SUP-1602 Snyk task API migration from deprecated version to REST API version

### PR DESCRIPTION
**Pull request by jesutorrr**
[Snyk Api base url updated](https://kennasecurity.atlassian.net/browse/SUP-1602)

Problem

Due to changes in Snyk's API, customers now need to specify their API environment. Previously, we used the static URL:
`HOST = "https://snyk.io"`

Solution

We should request that customers provide their unique API_BASE_URL as a parameter during task execution. The API_BASE_URL should be one of the following valid values:
snyk_api_base:{
api.snyk.io
api.eu.snyk.io
api.au.snyk.io}

This base URL should be entered without the protocol (https://) and rest parts, as these will be appended automatically in the code. The complete URL will follow this format:

`https://#{snyk_api_base}/rest`

The task was updated to map the new schema showed in API documentation https://apidocs.snyk.io/?version=2024-04-29

Example of the task execution 
`docker run -it --rm -v ~/Desktop/toolkit_input:/opt/app/toolkit/input -v ~/Desktop/toolkit_output:/opt/app/toolkit/output -t kennasecurity/toolkit:latest task=snyk_v2 snyk_api_token=XXXXXXXXXXXXXXXXX snyk_api_base=api.eu.snyk.io`

image

Output file:
image

Vulns uploaded to CVM
image

### CODE REVIEW

File: tasks/connectors/snyk_v2/lib/snyk_v2_client.rb [hunk 1]
```diff
@@ -6,21 +6,20 @@ module SnykV2
       class SnykV2Client
         class ApiError < StandardError; end
 
-        HOST = "https://snyk.io"
-
-        def initialize(token)
+        def initialize(token, api_base_url)
           @token = token
+          @api_base_url = "https://#{api_base_url}/v1"
```
Comment (dianaroz) [inline tasks/connectors/snyk_v2/lib/snyk_v2_client.rb:11]: having api_base_url being just the base in snyk_v2_task and referring to the whole constructed url in snyk_v2_client is confusing and makes the code harder to read, can we change this to be @host or just @api_url ?

File: tasks/connectors/snyk_v2/readme.md [hunk 2]
```diff
@@ -36,4 +37,5 @@ Complete list of Options:
 | kenna_connector_id | false | If set, we'll try to upload to this connector | n/a |
 | kenna_api_key | false | Kenna API Key | n/a |
 | kenna_api_host | false | Kenna API Hostname | api.kennasecurity.com |
+| Snyk_api_url | true | Snyk environment API base URL without prefix e.g. api.eu.snyk.io, api.snyk.io or api.au.snyk.io | n/a  |
```
Comment (dianaroz) [inline tasks/connectors/snyk_v2/readme.md:40]: - parameter name in snyk_v2_task.rb is snyk_api_base
- parameters are case sensitive when running the toolkit image so we should use all lowercase in readme file to match our definitions

**Review by dianaroz — CHANGES_REQUESTED** [review]

**Review by BlakeHendersonKS — CHANGES_REQUESTED** [review]
Can we try changing 
```
            { name: "snyk_api_base",
              type: "string",
              required: true,
              default: nil,
              description: "Snyk environment API base URL without prefix e.g. api.eu.snyk.io or api.snyk.io or api.au.snyk.io" }
 ```
 to be optional and default to the US otherwise? I think ought to clear up the test failure.

File: tasks/connectors/snyk_v2/readme.md (hunk 2 above)
Comment (jesutorrr) [reply tasks/connectors/snyk_v2/readme.md]: Task updated with change, thanks!

**Comment by slblack** [comment]
[snyk_kdi_vulns_batch_1.json](https://github.com/user-attachments/files/15510734/snyk_kdi_vulns_batch_1.json)

Am getting this connector error when adding the data to the clients subscription.

Exception occurred: NoMethodError - undefined method `scrub' for #<Hash:0x00007f907504e8d8> - ["/opt/apps/importer/lib/importer/payload/asset_sanitizer.rb:31:in `block in sanitize_vulns'",
 "/opt/apps/importer/lib/importer/payload/asset_sanitizer.rb:30:in `each'",
 "/opt/apps/importer/lib/importer/payload/asset_sanitizer.rb:30:in `sanitize_vulns'",
 "/opt/apps/importer/lib/importer/payload/asset_sanitizer.rb:21:in `block in call'",
 "/opt/apps/importer/lib/importer/profiler.rb:22:in `call'",
 "/opt/apps/importer/lib/importer/payload/asset_sanitizer.rb:19:in `call'",
 "/opt/apps/importer/lib/importer/kenna/creates_payload/sanitizes_action.rb:10:in `block in <class:SanitizesAction>'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:25:in `block (4 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `catch'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `block (3 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/context/key_verifier.rb:49:in `verify_keys'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:20:in `block (2 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:14:in `block in with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `block in measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/span_operation.rb:152:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:376:in `start_span'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:159:in `block in trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/context.rb:45:in `activate!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:158:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing.rb:18:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:13:in `with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:19:in `block in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:60:in `block in invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:19:in `call'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:56:in `invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:31:in `block in reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each_with_object'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer_log_decorator.rb:33:in `reduce'",
 "/opt/apps/importer/lib/importer/kenna/creates_payload/organizer.rb:11:in `call'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:58:in `block in invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:19:in `call'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:56:in `invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:31:in `block in reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each_with_object'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer_log_decorator.rb:33:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/scoped_reducable.rb:6:in `scoped_reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_callback.rb:14:in `block (2 levels) in run'",
 "/opt/apps/importer/lib/importer/kenna/iterates_assets_action.rb:11:in `block (2 levels) in <class:IteratesAssetsAction>'",
 "/opt/apps/importer/lib/importer/kenna/iterates_assets_action.rb:9:in `each'",
 "/opt/apps/importer/lib/importer/kenna/iterates_assets_action.rb:9:in `block in <class:IteratesAssetsAction>'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:25:in `block (4 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `catch'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `block (3 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/context/key_verifier.rb:49:in `verify_keys'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:20:in `block (2 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:14:in `block in with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `block in measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/span_operation.rb:152:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:376:in `start_span'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:159:in `block in trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/context.rb:45:in `activate!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:158:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing.rb:18:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:13:in `with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:19:in `block in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_callback.rb:18:in `block in run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:58:in `block in invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:19:in `call'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:56:in `invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:31:in `block in reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each_with_object'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer_log_decorator.rb:33:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/scoped_reducable.rb:6:in `scoped_reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_callback.rb:14:in `block (2 levels) in run'",
 "/opt/apps/importer/lib/importer/shared/iterates_client_files_action.rb:12:in `block (2 levels) in <class:IteratesClientFilesAction>'",
 "/opt/apps/importer/lib/importer/shared/iterates_client_files_action.rb:10:in `each'",
 "/opt/apps/importer/lib/importer/shared/iterates_client_files_action.rb:10:in `block in <class:IteratesClientFilesAction>'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:25:in `block (4 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `catch'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:23:in `block (3 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/context/key_verifier.rb:49:in `verify_keys'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:20:in `block (2 levels) in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:14:in `block in with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `block in measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/span_operation.rb:152:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:376:in `start_span'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:159:in `block in trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/context.rb:45:in `activate!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:158:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing.rb:18:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/profiling/action.rb:13:in `with_profiling'",
 "/opt/apps/bundle/ruby/2.7.0/gems/beehive-2.70.5/lib/beehive/light_service_patches/action.rb:19:in `block in executed'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_callback.rb:18:in `block in run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:58:in `block in invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:19:in `call'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:56:in `invoke_action'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:31:in `block in reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `each_with_object'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer.rb:29:in `reduce'",
 "/opt/apps/bundle/ruby/2.7.0/gems/light-service-0.12.0/lib/light-service/organizer/with_reducer_log_decorator.rb:33:in `reduce'",
 "/opt/apps/importer/lib/importer/kenna/organizer.rb:7:in `call'",
 "/opt/apps/importer/lib/importer.rb:47:in `block (3 levels) in run'",
 "/opt/apps/importer/lib/importer.rb:59:in `block (2 levels) in run'",
 "/opt/apps/importer/lib/importer/utility/monitoring.rb:46:in `duration'",
 "/opt/apps/importer/lib/importer.rb:44:in `block in run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `block in measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/span_operation.rb:152:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:376:in `start_span'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:159:in `block in trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/context.rb:45:in `activate!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:158:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing.rb:18:in `trace'",
 "/opt/apps/importer/lib/importer.rb:21:in `run'",
 "/opt/apps/importer/lib/importer_worker.rb:11:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-status-0.4.1/lib/resque/plugins/status.rb:148:in `safe_perform!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-status-0.4.1/lib/resque/plugins/status.rb:124:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/job.rb:181:in `block (3 levels) in perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/contrib/resque/resque_job.rb:53:in `block in around_perform0_ddtrace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `block in measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/span_operation.rb:152:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/trace_operation.rb:171:in `measure'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:376:in `start_span'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:159:in `block in trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/context.rb:45:in `activate!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/tracer.rb:158:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing.rb:18:in `trace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/contrib/resque/resque_job.rb:38:in `around_perform0_ddtrace'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/job.rb:180:in `block (2 levels) in perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/job.rb:188:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-resque-5.11.0/lib/sentry/resque.rb:10:in `block in perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-resque-5.11.0/lib/sentry/resque.rb:31:in `block in record'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-ruby-5.11.0/lib/sentry/hub.rb:59:in `with_scope'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-ruby-5.11.0/lib/sentry-ruby.rb:365:in `with_scope'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-resque-5.11.0/lib/sentry/resque.rb:20:in `record'",
 "/opt/apps/bundle/ruby/2.7.0/gems/sentry-resque-5.11.0/lib/sentry/resque.rb:9:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/ddtrace-1.2.0/lib/datadog/tracing/contrib/resque/resque_job.rb:20:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:304:in `perform'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:894:in `block in perform_with_fork'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:892:in `fork'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:892:in `perform_with_fork'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:260:in `work_one_job'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:234:in `block in work'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:231:in `loop'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resque-1.27.4/lib/resque/worker.rb:231:in `work'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/worker.rb:94:in `try_start'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:214:in `block in start_idle_workers'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:212:in `each'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:212:in `start_idle_workers'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:103:in `block in run_workers_run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:100:in `loop'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:100:in `run_workers_run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:84:in `run'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/lib/resqued/listener.rb:62:in `exec!'",
 "/opt/apps/bundle/ruby/2.7.0/gems/resqued-0.8.6/exe/resqued:5:in `<top (required)>'",
 "/usr/local/bin/resqued:23:in `load'",
 "/usr/local/bin/resqued:23:in `<main>'"]

**Comment by jesutorrr** [comment]
@slblack Please try with the new version I sent you.

**Pull request merged**